TKeazirian.HTTPServer/Request/RequestParser.cs

```diff
@@ -0,0 +1,45 @@
+namespace TKeazirian.HTTPServer.Request;
+
+public class RequestParser
+{
+    public Request ParseRequest(string clientRequest)
+    {
+        string requestMethod = ParseRequestMethod(clientRequest);
+        string requestPath = ParseRequestPath(clientRequest);
+        string requestHeaders = ParseHeaders(clientRequest);
+        string? requestBody = ParseRequestBody(clientRequest);
+
+        return new Request(requestMethod, requestPath, requestHeaders, requestBody);
+    }
+
+
+    public string ParseRequestMethod(string requestString)
+    {
+        string[] requestArray = requestString.Split(Constants.Space, 2);
+        return requestArray[0];
+    }
+
+    public string ParseRequestPath(string requestString)
+    {
+        string[] requestArray = requestString.Split(Constants.Space, 3);
+        return requestArray[1];
+    }
+
+    public string ParseHeaders(string requestString)
```

Comment:
Nitpick: the rest of the methods in this class follow a `ParseRequest___` pattern, but this one doesn't